hs_csdms/models.py

```diff
@@ -0,0 +1,69 @@
+from django.db import models, transaction
+from .utils import split_csdms_names
+
+
+class CSDMSName(models.Model):
+    """ Maintain a cached copy of the CSDMS standard name database inside HydroShare
+        This maintains a copy of the CSDMS database maintained at
+        https://csdms.colorado.edu/wiki/CSDMS_Standard_Names
+    """
+    source = models.CharField(max_length=256, editable=False, null=False, blank=False,
+                              help_text='The source file from CSDMS')
+    part = models.CharField(max_length=16, choices=(('name', 'name'), ('decoration', 'decoration'),
+                            ('measure', 'measure')),
+                            help_text='The standard names can be splitted to three parts,\
+                                       name, decoration, and measurement')
+    value = models.CharField(max_length=255, editable=False, null=False, blank=False,
+                             help_text='The value for the corresponding part')
+
+    @staticmethod
+    def add_word(s, p, v):
+        """ add a record to the CSDMS Database
+            :param s, a choice from source describes where the word from
+            :param p, a choice from part describes which part the word is extracted from
+            :param, v, a string value for the word to be added
+        """
+        with transaction.atomic():
+            object, _ = CSDMSName.objects.get_or_create(source=s, part=p, value=v)
+
+        return object
+
+    @staticmethod
+    def populate_csdms_names():
+        """ Store each part from split_csdms into LDAWord model
+        """
+        csdms_names, csdms_decors, csdms_measures = split_csdms_names()
```

Comment:
Isn't this already done in the migration here?
hs_csdms/migrations/0002_auto_20200626_2152.py

Reply:
Yes. This has been done in the migration file. This function is just used for helping manually populate the database, if necessary. Maybe, we could remove it.

Reply:
or use it in your migration

Reply:
You need a test around this method as well

Reply:
It looks like we can not call a class method in the migration file. 
https://stackoverflow.com/questions/28777338/django-migrations-runpython-not-able-to-call-model-methods
Maybe I misunderstood it. 
I will put a test case for it.

Reply:
Fair enough.  I'd argue that this doesn't belong in a migration at all because it could change.  I think I'd rather have it as a a management command that we can update the csdms names input as needed.

Reply:
What about both? The management command can update, and the migration can do the initial setting.